base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml

```diff
@@ -1398,6 +1398,16 @@
         <categories/>
       </value>
     </entry>
+    <entry>
+      <key>Riken PSU Status Summary</key>
+      <value>
+        <type>psu_status_summary</type>
+        <path>RIKEN\Magnet\psu_status_summary.opi</path>
```

Comment:
Slashes in this direction will cause this to fail on linux. Can you swap the slashes to forward slashes which work on both platforms?
```suggestion
        <path>RIKEN/Magnet/psu_status_summary.opi</path>
```